uphold-integration/src/main/java/org/dash/wallet/integration/uphold/data/UpholdClient.java

```diff
@@ -120,14 +141,37 @@ public void onResponse(Call<UpholdAccessToken> call, Response<UpholdAccessToken>
                     storeAccessToken();
                     getCards(callback, null);
                 } else {
-                    log.error("Error to obtain Uphold access token", response.message());
+                    log.error("Error to obtain Uphold access token " + response.message());
                     callback.onError(new Exception(response.message()), false);
```

Comment:
For the logs, we need to use %s in the strings or + to get the extra information included.  Many times the message is blank.